README.md

```diff
@@ -23,11 +23,11 @@ portFFT is in early stages of development and will support more options and opti
 
 Clone portFFT and run the following commands from the cloned repository.
 
-Build using DPC++ 2023.2.0 as:
+Build using DPC++ 2024.0 as:
 
 ```shell
-source /opt/intel/oneapi/compiler/2023.2.0/env/vars.sh
-cmake -Bbuild -DCMAKE_CXX_COMPILER=/opt/intel/oneapi/compiler/2023.2.0/linux/bin/icpx -DPORTFFT_BUILD_TESTS=ON -DPORTFFT_BUILD_BENCHMARKS=ON
+source /opt/intel/oneapi/compiler/2024.0/env/vars.sh
+cmake -Bbuild -DCMAKE_CXX_COMPILER=/opt/intel/oneapi/compiler/2024.0/linux/bin/icpx -DPORTFFT_BUILD_TESTS=ON -DPORTFFT_BUILD_BENCHMARKS=ON
```

Comment:
icpx and vars.sh have moved in my install of 2024.0. There is no `oneapi` and no `linux` folder.

Reply:
That's weird because I have the `oneapi` folder, I don't remember changing the install path but maybe it re-used the previous one. This could add some confusion so I've replaced the start of the path with `/path/to/intel/oneapi`, do you think that's fine?
I didn't notice `linux` was gone. I've checked that setting the variable to `icpx` alone is enough as long as it is in the `PATH`.

Reply:
Ok, maybe my issue is that I install it in a custom location and separately from previous installs. The changes look good to me.